src/components/main-view/messages.js

```diff
@@ -6,21 +6,13 @@ export default defineMessages({
     description: 'The page title of main view',
     defaultMessage: 'Hello, world',
   },
-  RestCallsLabelLink: {
-    id: 'MainView.RestCalls',
-    description: 'The label for the link to RestCalls',
-    defaultMessage: 'RestCalls',
-  },
-  GraphqlCallsLabelLink: {
-    id: 'MainView.GraphqlCalls',
-    description: 'The label for the link to GraphqlCalls',
-    defaultMessage: 'GraphqlCalls',
-  },
-  homepageLabelLink: {
-    id: 'MainView.homepageLabelLink',
-    description: 'The label for the link to homepage',
-    defaultMessage: 'Homepage',
+  
+  ShoppingListsLabelLink: {
```

Comment:
I would suggest to either define all messages like this or to completely skip the i18n part (so you always hardcode the message/string).

Otherwise it's kind of confusing.